src/key.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// The verification key (aka image id, the hash of the guest program)
+pub struct Vk(pub risc0_zkp::core::digest::Digest);
+
+impl From<[u32; 8]> for Vk {
```

Comment:
If this input should be the same length of a Digest, we might refer directly to the constant `risc0_zkp::core::digest::DIGEST_BYTES` instead of using the hardcoded value.

Reply:
Now using `risc0_zkp::core::digest::DIGEST_WORDS`.